packages/docs/src/pages/qwikcity/routing/pathless.mdx

```diff
@@ -0,0 +1,17 @@
+---
+title: Qwikcity - Pathless
+---
+
+# Qwikcity - Pathless
```

Comment:
Idea: This page could be merged with `_ prefix`-paragraph.

Reply:
I had the same thought, but if I merge it it will not show up in the menu on the right, and I think it is useful for people looking for things, so my current thinking is to leave as is.